prepare_input_files_lists/prepare_input_files_list.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
```

Comment:
Please, you cannot change the config file, this is what keeps Keane, Nukul and I in sync, can you revert all the changes made here?
If you want to have your own script, then it's better you create yours.